recipe_scrapers/gesundaktiv.py

```diff
@@ -8,32 +7,12 @@ class GesundAktiv(AbstractScraper):
     def host(cls):
         return "gesund-aktiv.com"
 
-    def site_name(self):
-        raise StaticValueException(return_value="gesund + aktiv")
-
-    def title(self):
-        return self.soup.find("h1", {"class": "page-header"}).get_text()
-
-    def image(self):
-        return self.soup.find("a", {"class": "colorbox"})["href"]
-
     def ingredients(self):
         ingredients_container = self.soup.find(
-            "div", {"class": "field--name-field-zutaten"}
+            "div", {"class": "news-recipes-indgredients"}
         )
-        ingredients = ingredients_container.findAll("div", {"class": "field--item"})
-        return [normalize_string(instruction.get_text()) for instruction in ingredients]
+        ingredients = ingredients_container.findAll("li")
+        return [normalize_string(ingredient.get_text()) for ingredient in ingredients]
 
-    def instructions(self):
-        instructions_container = self.soup.find(
-            "div", {"class": "field--name-field-content-element"}
-        )
-        instructions = instructions_container.findAll("div", {"class": "field--item"})
-        return "\n".join(
-            [normalize_string(instruction.get_text()) for instruction in instructions]
-        )
-
-    def description(self):
-        return normalize_string(
-            self.soup.find("meta", {"name": "description"})["content"]
-        )
+    def author(self):
+        return self.site_name()
```

Comment:
Let's also move this method to the top of the class definition?

(I'm not suggesting we enforce the same ordering as JSON... yet :))

Reply:
This may be a nice cleanup & I'd think pretty easy to implement!